spec/features/admin/admin_items_spec.rb

```diff
@@ -273,14 +273,14 @@
     end
 
     it 'can edit an item' do
-      visit admin_items_path(user_id: @merchant)
+      visit admin_items_path(slug: @merchant.slug)
 
       within "#item-#{@item_1.id}" do
         expect(page).to have_button("Edit Item")
         click_button("Edit Item")
       end
 
-      expect(current_path).to eq(edit_admin_item_path(@item_1))
+      expect(current_path).to eq(edit_admin_item_path(@item_1.slug))
```

Comment:
comparing this line to line 276 above, any thoughts on why you needed to include `(slug: ` above but didn't need to do it here?

Reply:
On line 276, `(slug: ` is as a query param (slug being for the merchant), to the admin/items path rather than a part of the path itself. Line 283 the path is `admin/items/:slug/edit` where slug is the item's slug. I opted for using the query param thinking it would be more clear. Is routing through admin/items/:slug (merchant slug) better practice?